workers/unity/spatialos.MobileClient.worker.json

```diff
@@ -21,14 +21,9 @@
         "MobileClient"
       ]
     },
-    "entity_interest": {
-      "range_entity_interest": {
-        "radius": 25
-      }
-    },
     "component_delivery": {
       "default": "RELIABLE_ORDERED",
-      "checkoutAllInitially": true
+      "checkoutAllInitially": false
```

Comment:
Is the `component_delivery` part required? Or can it be removed entirely?

Reply:
The docs say to keep it https://docs.improbable.io/reference/14.4/shared/authority-and-interest/interest/chunk-based-interest-cbi#turning-off-cbi